CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `jj branch track` now show conflicts if there are some.
 
+* There are now prebuilt binaries for `aarch64-linux-unknown-musl`.
+
```

Comment:
re: what Ilya said, maybe add one more small nit: "Note, these are cross compiled and currently untested. We plan on providing fully tested builds later once our CI system allows it." or something like that

Reply:
Done :+1: